advanced-tools/gdb.rst

```diff
@@ -324,3 +324,50 @@ auto-load safe. One way to achieve this is to add a line like the following
 to ``~/.gdbinit`` (edit the specific list of paths as appropriate)::
 
     add-auto-load-safe-path ~/devel/py3k:~/devel/py32:~/devel/py27
+
+
+GDB tips
+========
+
+Learning to use GDB effectively improves your chances of successfully
+debugging problems with Python's internals.  The tips here aren't yet
+organized in any particular way.
```

Comment:
```suggestion
Learning to use GDB effectively improves your chances of successfully
debugging problems with Python's internals.
```

I would just remove this sentence, since there's only two tips and they don't warrant any particular organization.
If we end up adding more tips, we can say something like "miscellaneous tips in no particular order" (which doesn't imply that they will be organized in the future).